src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Think about which variables should be local in the method and which should be class-level.

Reply:
> Think about which variables should be local in the method and which should be class-level.

Must avoid object creation in method (?)